backend/src/bundles/stripe-events/repositories/subscription-plan.repository.ts

```diff
@@ -32,10 +32,15 @@ class SubscriptionPlanRepository implements ISubscriptionPlanRepository {
             .returning('*');
     }
 
-    public async delete(id: string): Promise<boolean> {
+    public async delete({
+        stripePlanId,
+    }: {
+        stripePlanId: string;
+    }): Promise<boolean> {
         const response = await this.subscriptionPlanModel
             .query()
-            .deleteById(id);
+            .where({ stripePlanId })
+            .delete();
         return !!response;
```

Comment:
```suggestion
        return Boolean(response);
```
Using !! is not very readable.